pkg/persistence/config/writer/config_writer.go

```diff
@@ -791,6 +792,36 @@ func fmtDetailedConfigWriterError(context *serializerContext, format string, arg
 	}
 }
 
+var maxFileNameLen = 254
```

Comment:
No need for this line. The default value will provide `GetEnvValueInt` in line 803.

Reply:
if its set using `GetEnvValueInt` then yes. Other than that I'd rather have it inititialized to a reasonable value if that's ok for you.

Reply:
Actually, this doesn't need to be a variable outside the function anymore at all.